webserver/server/codechecker/server/api/store_handler.py

```diff
@@ -342,18 +342,20 @@ def addCheckerRun(session, command, name, tag, username,
 
         # Create entry for analyzer statistics.
         for analyzer_type, res in statistics.items():
-            version = res.get('version')
+            analyzer_version = res.get('version', None)
             successful = res.get('successful')
             failed = res.get('failed')
             failed_sources = res.get('failed_sources')
 
-            analyzer_version = None
-            if version:
+            if analyzer_version:
                 analyzer_version = zlib.compress(version,
                                                  zlib.Z_BEST_COMPRESSION)
 
             compressed_files = None
             if failed_sources:
+                if version == '6.9.0':
```

Comment:
I think this is a typo here:
```suggestion
                if analyzer_version == '6.9.0':
```

Reply:
No, this is not a typo. `analyzer_version` comes from the metadata.json and contains the git hash, but  we are using version which comes from the client.